cime_config/testdefs/testmods_dirs/clm/NEON/NIWO/shell_commands

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+./xmlchange BFBFLAG="TRUE"
```

Comment:
Maybe instead of having a specific testmod for NIWO we should add "default" to the include_user_mods to get this and other default settings. Maybe that should also go into the main NEON directory, and then each NEON subdirectory would use it.

Reply:
I have implemented this in cime https://github.com/ESMCI/cime/pull/4020

Reply:
@ekluzek The change that @billsacks  is making to cime will mean that the test name will look like
SMS_D_Vnuopc_Mmpi-serial.CLM_USRDAT.I1PtClm51Bgc.cheyenne_intel.clm-default--NEON-NIWO 

If you still think that you're comment is relevant let's talk and clarify.

Reply:
That is my preference to change the testname and bring in this with @billsacks update. Should we do a meeting with the three of us? @billsacks what  do you think here?

Reply:
I agree. I don't think we need another meeting unless you have some unresolved questions.

I got pulled into some other things, but I hope to have my cime PR ready tomorrow.

Reply:
Perfect. So we'll just need an update to cime in Externals with that change. And the testname needs to change. But, then this will be ready to go.

Reply:
When the cime tag is ready I will test and add the new testname to the testlist in this branch.

Reply:
This is complete.